devices/src/legacy/rtc_pl031.rs

```diff
@@ -107,8 +110,8 @@ impl RTC {
             }
             RTCLR => {
                 self.load = val;
-                self.previous_now = SteadyTime::now();
-                self.tick_offset = Timespec::new(i64::from(val), 0);
+                self.previous_now = Instant::now();
+                self.tick_offset = i64::from(val) * 1_000_000_000;
```

Comment:
`1_000_000_000` shows up several times as a magic value, please declare a `const` for it instead